tests/test_server.py

```diff
@@ -1,204 +1,98 @@
-import os
+import json
 from unittest.mock import patch
 
+from mks.server import application
 from tma_saml import FlaskServerTMATestCase
 from tma_saml.for_tests.cert_and_key import server_crt
 
-from mks.server import application
-
-FIXTURE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'fixtures')
-BSN_RESPONSE_PATH = os.path.join(FIXTURE_PATH, "hr_bsn_response.xml")
-KVK_RESPONSE_PATH = os.path.join(FIXTURE_PATH, "hr_kvk_prs_response.xml")
-RESPONSE_EMPTY_PATH = os.path.join(FIXTURE_PATH, "hr_empty_response.xml")
-
+from .test_mks_client_bsn_hr import BSN_HR_RESPONSE, get_bsn_xml_response_fixture
+from .test_mks_client_kvk_prs_hr import KVK_HR_PRS_RESPONSE, get_kvk_prs_xml_response_fixture, get_xml_response_empty_fixture
+from .test_mks_client_kvk_mac_hr import KVK_HR_MAC_RESPONSE, get_kvk_mac_xml_response_fixture
+from .test_mks_client_nnp_hr import NNP_HR_RESPONSE, get_nnp_xml_response_fixture
```

Comment:
Wees voorzichtig met relative imports, dit is prima, maar goed op blijven letter ermee.

Reply:
Reason being?

Reply:
Het zijn niet allemaal hele sterke redenen, maar bij elkaar vind ik het de moeite waard om ze niet al te vaak te gebruiken.

- Het kan problemen verbergen met PYTHONPATH
- Het is minder duidelijk waar iets vandaan komt
- Verplaatsen van code